test/e2e-common.sh

```diff
@@ -39,7 +39,26 @@ function knative_setup() {
   echo ">> Starting Knative Eventing"
   echo "Installing Knative Eventing"
   ko apply -f ${EVENTING_CONFIG} || return 1
+
+  # Force tracing to 100%.
+  kubectl apply -f - << END
```

Comment:
Why do we set the configmap both here and in the test?

Reply:
Removed.